bin/newman.js

```diff
@@ -80,6 +80,12 @@ program
         });
     });
 
+program.on('--help', function () {
+    console.info('');
+    console.info('To get available options for a command:');
```

Comment:
The above `console.info` statement can be removed if this one is prefixed with `\n`, like so:
```
console.info('\nTo...');
```